puppet/modules/slave/manifests/init.pp

```diff
@@ -168,6 +168,8 @@
       ensure => present;
     'ansible':
       ensure => latest;
+    'google-chrome-stable':
```

Comment:
I just realized that this will break on Debian since the package is not available there. You could install chromium but that might result in different paths.